src/TulsiGenerator/Bazel/tulsi/tulsi_aspects.bzl

```diff
@@ -710,6 +710,16 @@ def _tulsi_sources_aspect(target, ctx):
                         _collect_bundle_imports(rule_attr))
 
     copts_attr = _get_opt_attr(rule_attr, "copts")
+    if copts_attr:
+        expanded_attrs = []
+        for attr in copts_attr:
+            if '$(location' in attr:
```

Comment:
Does calling `ctx.expand_location` do anything wrong if it's used on something that _doesn't_ have a `$(location <target>)` template?

i.e. could you get rid of the `if` block and just do something like:

```
if copts_attr:
    copts_attr = [ctx.expand_location(attr, [target]) for attr in copts_attr]
```

It seems like that should work (existing tests all pass with that at least), but having a good example to use for an integration test would be helpful.

Reply:
Simplified the block as per your suggestion. That is working as expected.